lunar/distribution.yaml

```diff
@@ -1063,6 +1063,21 @@ repositories:
       url: https://github.com/ros-perception/perception_pcl.git
       version: lunar-devel
     status: maintained
+  pid:
+    doc:
+      type: git
+      url: https://bitbucket.org/AndyZe/pid.git
+      version: master
+    release:
+      tags:
+        release: release/kinetic/{package}/{version}
+      url: https://github.com/AndyZelenak/pid-release.git
+      version: 0.0.20-0
```

Comment:
@AndyZe did you generate this pull request with bloom ?
The version 0.0.20-0 doesn't match the one on the [release repository for lunar](https://github.com/AndyZe/pid-release#pid-lunar---0021-2) (which is 0.0.21-2). That is the reason why all the jobs are failing